### PR TITLE
Document chocolateyPackage* variables

### PR DESCRIPTION
**Pull request by mjiderhamn**
Adding documentation of the `chocolateyPackageFolder`, `chocolateyPackageName` and `chocolateyPackageVersion` environment variables made available to `chocolateyInstall.ps1` and `chocolateyUninstall.ps1`.

### CODE REVIEW

**Comment by Redsandro** [comment]
Seems good.

**Comment by gep13** [comment]
@Redsandro thanks for pulling this in, you did miss out one step though :-)

This repo isn't "connected" to the actual wiki, as GitHub doesn't make that easy.  We found out that you could use an additional repo, i.e. this one, as a conduit to the actual wiki, but it means you need to push to two places.  This was thanks to the guys over at TypeScript:

https://github.com/Microsoft/TypeScript-wiki/issues/2

There is an outstanding task on me here:

https://github.com/chocolatey/choco-wiki/issues/1

To document this properly.

File: HelpersReference.md [hunk 1]
```diff
@@ -117,4 +117,27 @@ These are the functions from above as one list.
...
+
+* __chocolateyPackageFolder__ = the folder where Chocolatey has downloaded and extracted the NuGet gallery.
+* __chocolateyPackageName__ (since 0.9.9) = The package name, which is equivalent to the `<id>` tag in the nuspec 
+* __chocolateyPackageVersion__ (since 0.9.9) = The package version, which is equivalent to the `<version>` tag in the nuspec
+ 
+`chocolateyPackageVersion` may be particularly useful, since that would allow you in some cases to create packages for new releases of the updated software by only changing the `<version>` in the nuspec and not having to touch the `chocolateyInstall.ps1` at all. An example of this:
+```
+$url = "http://www.thesoftware.com/downloads/thesoftware-$env:chocolateyPackageVersion.zip"
+
+Install-ChocolateyZipPackage '$chocolateyPackageName' $url $binRoot
```
Comment (gep13) [inline HelpersReference.md]: @mjiderhamn shouldn't this be $env:chocolateyPackageName

Comment (mjiderhamn) [inline HelpersReference.md]: Indeed - thanks! I've updated my branch at https://github.com/mjiderhamn/choco-wiki/commit/b04b5564c8b9b929b4f0345741e709daa659d640 but I guess this PR is not updated since it is closed. Will you re-open or do I need to create another PR?

**Comment by Redsandro** [comment]
Please create another PR - after a merge it is "closed" by design. Or rather "done".

**Comment by Redsandro** [comment]
@gep13 I'm not sure about that step. I thought the merge made it so?

Also does this not overwrite manual changes to the wiki? (E.g. I edited the rules recently).

**Comment by gep13** [comment]
@Redsandro there are now two repo's:

chocolatey\choco.wiki

and:

chocolatey\choco-wiki

choco.wiki is the default repository behind any GitHub repository, and where all the changes are stored when you edit the wiki through the GitHub site.

choco-wiki is a direct copy of choco.wiki, and allows us to accept PR's for documentation changes, but we need to take ownership of pushing/merging the changes into both choco.wiki and choco-wiki.

Yes, we will have to be careful of direct changes that have been made to choco.wiki when we are merging into choco.wiki from choco-wiki, but that is something we will just have to be careful about.

As for making manual changes to the wiki through the GitHub interface, this will still be allowed and supported, but only by contributors to choco, all other documentation changes by the community will be done through PR's.
